app/src/androidTest/java/org/openobservatory/ooniprobe/ui/measurements/CircumventionTest.java

```diff
@@ -0,0 +1,175 @@
+package org.openobservatory.ooniprobe.ui.measurements;
+
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.test.core.app.ActivityScenario;
+import androidx.test.core.app.ApplicationProvider;
+
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.openobservatory.ooniprobe.R;
+import org.openobservatory.ooniprobe.ResultFactory;
+import org.openobservatory.ooniprobe.activity.ResultDetailActivity;
+import org.openobservatory.ooniprobe.model.database.Measurement;
+import org.openobservatory.ooniprobe.model.database.Result;
+import org.openobservatory.ooniprobe.test.suite.CircumventionSuite;
+import org.openobservatory.ooniprobe.utils.DatabaseUtils;
+import org.openobservatory.ooniprobe.utils.FormattingUtils;
+
+import tools.fastlane.screengrab.locale.LocaleTestRule;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+import static org.openobservatory.ooniprobe.ui.measurements.UIMeasurementsUtils.assertMeasurementHeader;
+import static org.openobservatory.ooniprobe.ui.measurements.UIMeasurementsUtils.assertMeasurementOutcome;
+import static org.openobservatory.ooniprobe.ui.measurements.UIMeasurementsUtils.assertMeasurementRuntimeAndNetwork;
+
+public class CircumventionTest {
+
+    private static final String TEST_RESULTS_NOT_AVAILABLE = "N/A";
+    private static final String TEST_RESULTS_AVAILABLE = "OK";
```

Comment:
Ideally we should fetch this from `strings`. Otherwise, if the string is changed, the test starts failing.